android/clientlib-ktx/build.gradle

```diff
@@ -33,6 +35,84 @@ android {
     buildFeatures {
         buildConfig = false
     }
+
+    publishing {
+        singleVariant('release') {
+            withSourcesJar()
+            withJavadocJar()
+        }
+    }
+}
+
+publishing {
+    publications {
+        release(MavenPublication) {
+            groupId = group
+            artifactId = 'mobile-wallet-adapter-clientlib-ktx'
+            pom {
+                name = 'Mobile Wallet Adapter - Client library'
```

Comment:
We should probably update the `name` and `description` fields to be different than `clientlib`

Reply:
Done.